src/Common/src/Interop/Windows/Advapi32/Interop.GetSecurityInfoByHandle.cs

```diff
@@ -9,9 +9,16 @@ internal static partial class Interop
 {
     internal static partial class Advapi32
     {
-        [DllImport(Interop.Libraries.Advapi32, EntryPoint = "GetSecurityInfo", CallingConvention = CallingConvention.Winapi,
+        [DllImport(Libraries.Advapi32, EntryPoint = "GetSecurityInfo", CallingConvention = CallingConvention.Winapi,
```

Comment:
@JeremyKuhne is there anything I should modify from these P/Invokes?

Reply:
CallingConvention isn't needed, it is the default.

Reply:
And to match our existing patterns we should name these the same as the underlying API.

Reply:
OK I'll rename them and will remove CallingConvention.

BTW The tests are failing due to the long paths causing problems.